### DISCUSSION TITLE
configured callback hostname not shown in "Re-Configure the CCU" dialogue

### DISCUSSION THREAD

**Discussion by dotlambda** [SukramJ/aiohomematic#1645] [General]
As per https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port the "Callback Hostname/IP-Address" field is always empty when reconfiguring. Why? Doesn't it make sense to show the configured values?

**Comment by SukramJ** [comment]
> Why?

Nobody cares. Nobody fixed it.

> Doesn't it make sense to show the configured values?

Absolutely

**Reply by SukramJ** [reply]
> Why?

As far as I can remember, there was a problem with the optional fields and default values. Then you can no longer empty the fields if you don't need the values.
Someone needs to take care of that.
Feel free to take the task and create a pull request.

**Reply by dotlambda** [reply]
Here's the PR: https://github.com/danielperna84/custom_homematic/pull/706
